frontend/src/router.js

```diff
@@ -39,6 +40,7 @@ const routes = [
   { path: '/gems/comparison', name: 'comparemodels', component: CompareModels },
   { path: '/resources', name: 'resources', component: Resources },
   { path: '/documentation', name: 'documentation', component: Documentation },
+  { path: '/external-db/:dbName/:externalId', name: 'externalDb', component: ExternalDb },
```

Comment:
Thinking more about this, it would make sense to rephrase "external-db" to "identifier", at least in all places that the user sees. At least I'm not seeing the value in specifying "external", as it can lead to confusion (eg "external to whom?").

Reply:
Good suggestion although I'm not sure about having `identifier` as the route name. I personally might prefer something like `db-reference`. Would be good to get some more opinion on this as well. @inghylt @nanjiangshu

Reply:
I'm up for changing it! I think there is a value to somehow emphasize that the id:s are from separate databases so users understands what's metatlas specific and what's not. So would say I'm more in favour of `db-reference` or similar

Reply:
I see all models as independent of Metabolic Atlas, so I don't see much of a difference between what is "internal" and what is "external" - all of the source files are (at least supposed to) come straight from the model repository.

Reply:
How about `database-reference` or `database-identifier`?

Reply:
Ah, I see. I feel like the models are very close in relation to metatlas compared to the other databases, but I get what you mean. I would go for `database-identifier`, feels more in line with the usage of ID etc. on the web page

Reply:
Suggestion: how about only `identifiers`? Then we kind of mimic identifiers.org which me and Nanjiang think makes sense

Reply:
It would make sense to me too, it's along the lines of what I suggested initially.

Reply:
Created issue #608, resolving conversation